### PR TITLE
fix: nft plugin link font style and 52years

### PR DESCRIPTION
**Pull request by BillySh1**
## Description

Closes # MF-1825 MF-1822 MF-1823

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/web3-shared/base/src/utils/timestamp.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export function isValidTimesTamp(time?: number) {
```
Comment (UncleBill) [inline packages/web3-shared/base/src/utils/timestamp.ts]:
```suggestion
export function isValidTimestamp(time?: number) {
```

**Review by UncleBill — APPROVED** [review]

File: packages/web3-shared/base/src/utils/timestamp.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+    if (!time) return
+    return new Date(time).toString() !== new Date(1).toString()
```
Comment (UncleBill) [inline packages/web3-shared/base/src/utils/timestamp.ts]: After Ethereum's release date.
```suggestion
    return time >= new Date(2015, 6, 30).getTime()
```

Comment (BillySh1) [reply packages/web3-shared/base/src/utils/timestamp.ts]: before 2010-1-1 is inValid

File: packages/web3-shared/base/src/utils/timestamp.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const errorTriggerTimestamp = '2010-1-1'
```
Comment (guanbinrui) [inline packages/web3-shared/base/src/utils/timestamp.ts]:
```suggestion
const errorTriggerTimestamp = 1262304000000 // 2010-01-01
```

File: packages/web3-shared/base/src/utils/timestamp.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+const errorTriggerTimestamp = 1262304000000 // 2010-01-01
```
Comment (UncleBill) [inline packages/web3-shared/base/src/utils/timestamp.ts:1]: Name it 'ethereum genesis block date'

File: packages/web3-shared/base/src/utils/timestamp.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+
+export function isValidTimestamp(time?: number) {
+    if (!time) return
+    return time >= new Date(errorTriggerTimestamp).getTime()
```
Comment (UncleBill) [inline packages/web3-shared/base/src/utils/timestamp.ts]:
```suggestion
    return time >= ETHEREUM_GENESIS_BLOCK_DATE
```

File: packages/web3-shared/base/src/utils/timestamp.ts (hunk 4 above)
Comment (UncleBill) [reply packages/web3-shared/base/src/utils/timestamp.ts:1]: https://etherscan.io/block/0

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**